package-lock.json

```diff
@@ -24,6 +24,34 @@
         "js-tokens": "^4.0.0"
       }
     },
+    "@testim/chrome-version": {
+      "version": "1.0.7",
+      "resolved": "https://artifactory.engtools.xoom.com:443/artifactory/api/npm/npmjs-virtual/@testim/chrome-version/-/chrome-version-1.0.7.tgz",
+      "integrity": "sha1-DNkVeF7EGQ8Io6asybYfw4+18ak=",
+      "dev": true
+    },
+    "@types/glob": {
+      "version": "7.1.2",
+      "resolved": "https://artifactory.engtools.xoom.com:443/artifactory/api/npm/npmjs-virtual/@types/glob/-/glob-7.1.2.tgz",
```

Comment:
change registry to public npm (for all package-lock.json changes)

Reply:
@skratchdot I just updated these references to the public registry